tests/unit/development/metadata_test.py

```diff
@@ -4,46 +4,46 @@
 
 
 class Table(Component):
-    """A Table component.
-This is a description of the component.
-It's multiple lines long.
```

Comment:
Don't touch this file, as it's intended to match the output of our component generators. Unless you want to fix the component generator to match - but I'd prefer you not, in fact, as we have some other work in the wings to update that. Anyway that's a bigger project...